lemmatized_text/views.py

```diff
@@ -47,7 +47,13 @@ def create(request):
         lemmatized_text.clone(cloned_by=request.user)
         return redirect("lemmatized_texts_list")
 
-    return render(request, "lemmatized_text/create.html", {"select_lang": select_lang})
+    lang_dict = []
+    for lang in settings.SUPPORTED_LANGUAGES:
+        lang_dict.append({
+            "id": lang[0], "name": lang[1]
+        })
```

Comment:
Rather than mutating the SUPPORTED_LANGUAGES, you could pass it directly to the template:
```py
return render(request, "lemmatized_text/create.html", {"select_lang": select_lang, "supported_lang": settings.SUPPORTED_LANGUAGES})
```

And then in the template:
```
{% for lang in supported_lang %}
    <option value="{{ lang.0 }}" {% if select_lang == lang.0 %}selected{% endif %} >{{ lang.1 }}</option>
{% endfor %}
```
You can access the list values by `.0` and `.1` in the template.

Reply:
oh cool! Will make the changes now.